client/src/cmdhffelica.c

```diff
@@ -2661,44 +2907,48 @@ static int CmdHFFelicaAuthenticationLite(const char *Cmd) {
                   "Authenticate",
                   "hf felica liteauth -i 11100910C11BC407\n"
                   "hf felica liteauth -k 46656c69436130313233343536616263\n"
+                  "hf felica liteauth -k 46656c69436130313233343536616263 -@\n"
                   "hf felica liteauth -c 701185c59f8d30afeab8e4b3a61f5cc4 -k 46656c69436130313233343536616263"
                  );
     void *argtable[] = {
         arg_param_begin,
         arg_str0("k", "key", "<hex>", "set card key, 16 bytes"),
         arg_str0("c", "", "<hex>", "set random challenge, 16 bytes"),
         arg_str0("i", "", "<hex>", "set custom IDm"),
+        arg_lit0("@", "", "keep signal field ON after receive"),
```

Comment:
We have use `-k`  for keeping field ON else in the pm3 client.

Change `-@` to `-k`